examples/benchmark/tls_bench.c

```diff
@@ -105,6 +105,14 @@ bench_tls(args);
     #endif
 #endif
 
+#ifdef WOLFSSL_DTLS
+    #ifdef BENCH_EMBEDDED
+        #define TEST_DTLS_PACKET_SIZE   (2 * 1024 - 100)
```

Comment:
Are there any macros you can use for the `-100`? If not can you document the calc to get it?

Reply:
`-100` comes from [MAX_UDP_SIZE](https://github.com/wolfSSL/wolfssl/blob/7fabd74a90c383c44c5c2a93c25899ecba744936/wolfssl/internal.h#L1249). Removed `-100`, and put comments that where the value comes from.